veles/secrets/postmanapikey/validator.go

```diff
@@ -0,0 +1,236 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package postmanapikey contains validators for Postman API credentials.
```

Comment:
Only one package comment needs to be placed for the package - since this package has 3 files and there's already a comment in postmanapikey.go there's no need for another one in e.g. detector.go and validator.go